#! /bin/sh /usr/share/dpatch/dpatch-run
## 09_fix-minusign.dpatch by William Vera <billy@billy.com.mx>
##
## DP: Fix a hyphen-used-as-minus-sign in a manpage.

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' scrot-0.8~/scrot.1 scrot-0.8/scrot.1
--- scrot-0.8~/scrot.1	2010-07-07 14:24:51.000000000 -0500
+++ scrot-0.8/scrot.1	2010-07-07 14:41:32.000000000 -0500
@@ -87,7 +87,7 @@
 \\n prints a newline (ignored when used in the filename)
 .fi
 .SH EXAMPLE
-scrot '%Y-%m-%d_$wx$h.png' -e 'mv $f ~/shots/'
+scrot '%Y-%m-%d_$wx$h.png' \-e 'mv $f ~/shots/'
 .br
 This would create a file called something like
 2000-10-30_2560x1024.png and move it to your shots directory.
